apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -146,6 +165,21 @@ const CourseCalendarEvent = (props: CourseCalendarEventProps) => {
 
         const buildingId = locationIds[buildingName] ?? 69420;
 
+        let finalExamString = '';
```

Comment:
Same sentiment as the previous comment, it's confusing why we're changing the final exam logic on a PR that's supposed to be about supporting a new API time format. I'm assuming this version of the API returns the final as an object instead of a string now, which is the reason for this change, but make sure to mention that in the PR description.

Reply:
Gotcha, I'll add it into the PR!